worker-jobtracking/src/test/java/com/hpe/caf/worker/jobtracking/JobTrackingWorkerFactoryTest.java

```diff
@@ -30,14 +30,15 @@
 import org.junit.Test;
 import org.junit.runner.RunWith;
 import org.mockito.Mockito;
+
+import static junit.framework.TestCase.assertEquals;
 import static org.mockito.Mockito.verify;
 import org.mockito.runners.MockitoJUnitRunner;
 
 import java.util.Collections;
 import java.util.Date;
 import java.util.HashMap;
 import java.util.Map;
-import static org.junit.Assert.assertEquals;
```

Comment:
Why are we changing from `org.junit.Assert.assertEquals` to `junit.framework.TestCase.assertEquals` in this module?

Reply:
sorry -> removed